selectel/resource_selectel_mks_cluster_v1.go

```diff
@@ -0,0 +1,560 @@
+		},
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"kube_version": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"project_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"region": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					ru1Region,
+					ru2Region,
+					ru3Region,
+					ru7Region,
+					ru8Region,
+				}, false),
+			},
+			"nodegroups": {
+				Type:     schema.TypeSet,
+				Required: true,
+				ForceNew: false,
+				Set:      hashNodegroups,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"availability_zone": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"count": {
+							Type:     schema.TypeInt,
+							Required: true,
+						},
+						"keypair_name": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"affinity_policy": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"cpus": {
+							Type:     schema.TypeInt,
+							Optional: true,
+						},
+						"ram_mb": {
+							Type:     schema.TypeInt,
+							Optional: true,
+						},
+						"volume_gb": {
+							Type:     schema.TypeInt,
+							Optional: true,
+							Computed: true,
+						},
+						"volume_type": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+						},
+						"local_volume": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Computed: true,
+						},
+						"flavor_id": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+						},
+						"cluster_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"created_at": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"updated_at": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"nodes": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"id": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									"created_at": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									"updated_at": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									"hostname": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									"ip": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									"nodegroup_id": {
```

Comment:
Same as `cluster_id`. We don't need to store it since `nodes` is already nested in `nodegroups`.